include/revng/Model/ModelGraphPrinter.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Model/QualifiedType.h"
+
+/// Use this class to print a graph representing one or more model types
+/// and their subtypes.
```

Comment:
I'm impressed by your discipline in documentation!

Reply:
thanks!